### PR TITLE
Improve styles of Tables

### PR DESCRIPTION
**Pull request by viktorrenkema**
This PR changes some of the styling of Tables to make them feel more up to date.
- we use the `tint` if provided by the customization settings
- we inherit the rounded or sharp angles if provided by customization

# Screenshots
Examples of sample and existing public docs in light and dark mode

## Before
Keeper:
CleanShot 2024-12-11 at 21 56 06@2x

Snyk:
CleanShot 2024-12-11 at 21 57 34@2x

## After
Keeper:
CleanShot 2024-12-11 at 21 53 41@2x

Snyk:
CleanShot 2024-12-11 at 21 53 49@2x

Snyk dark:
CleanShot 2024-12-11 at 21 54 15@2x

Local, sharp radius set in customization settings:
CleanShot 2024-12-11 at 21 53 55@2x

### CODE REVIEW

**Review by zenoachtig — APPROVED** [review]
Looks good, nice job using `tint`!

**Review by scazan — APPROVED** [review]
code looks good. assuming the styles were tested across mobile and desktop and checked already. :)

File: packages/gitbook/src/components/DocumentView/Table/table.module.css [hunk 1]
```diff
@@ -23,21 +23,37 @@
 }
 
 :global(.dark) .tableWrapper {
-    @apply border-light/2;
+    @apply border-tint-50;
+}
+
+.columnHeader {
```
Comment (valentin0h) [inline packages/gitbook/src/components/DocumentView/Table/table.module.css:29]: Interesting name, `tableHeader` could be better perhaps.

**Review by valentin0h — APPROVED** [review]
nice 👌

**Comment by valentin0h** [comment]
> code looks good. assuming the styles were tested across mobile and desktop and checked already. :)

Or better check if there's visual tests for it and if not add one.

**Pull request merged**